backend/LexBoxApi/Services/ProjectService.cs

```diff
@@ -137,6 +137,26 @@ public async Task FinishReset(string code, Stream? zipFile = null)
         await dbContext.SaveChangesAsync();
     }
 
+    public async ValueTask<Guid[]> LookupProjectOrgIds(Guid projectId)
+    {
+        var cacheKey = $"ProjectOrgsForId:{projectId}";
+        if (memoryCache.TryGetValue(cacheKey, out Guid[]? orgIds) && orgIds != null) return orgIds;
+        orgIds = await dbContext.Projects
+            .Include(p => p.Organizations)
+            .Where(p => p.Id == projectId)
+            .Select(p => p.Organizations.Select(o => o.Id).ToArray())
+            .FirstOrDefaultAsync();
+        if (orgIds is null) return [];
```

Comment:
we should still cache null results I think.

Reply:
That would result in caching "no orgs" when a project doesn't exist, which then would mean remembering to invalidate the cache when the project is created otherwise for the first hour the org manager won't have access to it. But okay, as long as we don't forget to invalidate the cache then this would be fine. I just don't like that "as long as we don't forget" business: we've already gotten #781 because we forgot to invalidate a different cache...

Reply:
Done in commit 45dbafb7.